Cargo.toml

```
@@ -35,12 +35,12 @@ bitflags     = "1"
 futures-core = { version = "0.3.1", optional = true }
 inotify-sys  = "0.1.3"
 libc         = "0.2"
-tokio        = { version = "0.3.3", optional = true, features = ["net"] }
+tokio        = { version = "1.0.0", optional = true, features = ["net"] }
```

The `1.0.0` version is yanked.
```suggestion
tokio        = { version = "1.0.1", optional = true, features = ["net"] }
```

---

Thanks for the hint.